query-node/mappings/src/common.ts

```diff
@@ -5,8 +5,11 @@ import { Worker, Event, Network } from 'query-node/dist/model'
 import { BaseModel } from '@joystream/warthog'
 import { metaToObject } from '@joystream/metadata-protobuf/utils'
 import { AnyMetadataClass, DecodedMetadataObject } from '@joystream/metadata-protobuf/types'
+import BN from 'bn.js'
 
 export const CURRENT_NETWORK = Network.OLYMPIA
+
+export const INT32MAX = 2147483647
```

Comment:
I would just add comment to describe this as the max value the database can store of the `Int` type.

Reply:
Done in https://github.com/Joystream/joystream/pull/3299/commits/c7a129d856126767c1b2b43962de4728dddba3a2